### PR TITLE
Add formatting text sections

### PR DESCRIPTION
**Pull request by MARTINI1**
in Discord, Telegram and Slack wikis

### CODE REVIEW

File: Slack.md [hunk 1]
```diff
@@ -175,6 +176,18 @@ Below is an example of enabling the mini map for pokemon.
 | `maptype`      | Link to be added to notification text             | `roadmap`                                 |
 | `zoom`         | Specifies the zoom of the map                     | `15`                                      |
 
+### Formatting alarms text
+
+To customize the texts of the alarms here is a basic guide to apply styles to the text:
```
Comment (LegitDongo) [inline Slack.md]: Same as above:
`Here is a basic guide to apply custom styles to alarm text:`

File: Discord.md [hunk 2]
```diff
@@ -170,6 +171,23 @@ Below is an example of enabling the mini map for pokemon.
 | `maptype`      | Link to be added to notification text             | `roadmap`                                     |
 | `zoom`         | Specifies the zoom of the map                     | `15`                                          |
 
+### Formatting alarms text
+
+To customize the texts of the alarms here is a basic guide to apply styles to the text:
```
Comment (LegitDongo) [inline Discord.md]: This sounds strange in English,
Try this:
`Here is a basic guide to apply custom styles to alarm text:`

File: Telegram.md [hunk 3]
```diff
@@ -124,6 +125,17 @@ Below is how an advanced alarm configuration would appear in `alarms.json`. Note
 }
 ```
 
+### Formatting alarms text
+
+To customize the texts of the alarms here is a basic guide to apply styles to the text:
```
Comment (LegitDongo) [inline Telegram.md]: Same as above:
`Here is a basic guide to apply custom styles to alarm text:`

**Comment by MARTINI1** [comment]
changed those lines with @LegitDongo suggestion

thanks for review 👍

**Comment by kvangent** [comment]
Great addition

**Pull request merged**